app/code/Magento/ConfigurableProduct/Model/LinkManagement.php

```diff
@@ -10,6 +9,11 @@
 use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Framework\Exception\StateException;
 
+/**
+ * Configurable product link management.
+ *
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
Hi. Could you explain, please, why do we need `@SuppressWarnings` directive here? I understand that we have it in the original PR. However, we don't introduce a new dependency here and using this directive is a bit doubtful.